pytorch_lightning/core/lightning.py

```diff
@@ -1126,6 +1126,7 @@ def training_step(...):
 
         # backward
         self._running_manual_backward = True
+        loss /= self.trainer.accumulate_grad_batches
```

Comment:
I don't understand this addition. Can you explain?
Why does it not break any existing tests?

Reply:
Also making an assumption that we should average across accumulated batches right?

Reply:
We have the same assumption for `automatic optimization`.  I added this line to uniformize behaviours but the mean assumption makes more sense to me than the sum.

Reply:
ok but does this have to be in the lightningmodule? Wouldn't this go to the training loop where all the other accumulation logic is?

Reply:
Done, moved it internally.